cli/util/cancel.go

```diff
@@ -0,0 +1,72 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nspcc-dev/neo-go/cli/options"
+	"github.com/nspcc-dev/neo-go/cli/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/neorpc/result"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/actor"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+	"github.com/urfave/cli"
+)
+
+func cancelTx(ctx *cli.Context) error {
+	args := ctx.Args()
+	if len(args) == 0 {
+		return cli.NewExitError("transaction hash is missing", 1)
```

Comment:
Take a look at the Codecov labels in the Commits view:
![image](https://github.com/nspcc-dev/neo-go/assets/22775709/21aa6aff-2999-400f-8559-612a61c91468)
This line of code is not covered, although it can be done easily in the `TestUtilCancelTx`. Ditto for error cases below, we don't need to cover all of them, but at least some of them that are relatively easy to cover.